java-components/adhoc-analyser-tool/src/main/java/com/redhat/hacbs/analyser/kube/ResetArtifactBuildsCommand.java

```diff
@@ -31,15 +32,15 @@ public void run() {
             MixedOperation<ArtifactBuild, KubernetesResourceList<ArtifactBuild>, Resource<ArtifactBuild>> client = kubernetesClient
                     .resources(ArtifactBuild.class);
             if (!build.isEmpty()) {
-                ArtifactBuild item = client.withName(build).get();
-                item.getStatus().setState("");
-                client.updateStatus(item);
+                ArtifactBuild request = client.withName(build).get();
+                request.getMetadata().setAnnotations(Map.of("jvmbuildservice.io/rebuild", "true"));
+                client.createOrReplace(request);
             } else {
                 List<ArtifactBuild> items = client.list().getItems();
                 for (var request : items) {
                     if (!missing || request.getStatus().getState().equals("ArtifactBuildMissing")) {
-                        request.getStatus().setState("");
-                        client.updateStatus(request);
+                        request.getMetadata().setAnnotations(Map.of("jvmbuildservice.io/rebuild", "true"));
+                        client.createOrReplace(request);
```

Comment:
understanding this comment is not specifically related to this change, and quite possibly would be addressed in a separate PR, but currently we only give one shot for this api server invocation to pass or fail

intermittent / retryable errors are certainly part of k8s api server interaction - does the underlying client already handle that, or should we start incorporating that throughout the java component use if k8s client ?

Reply:
This is basically just a tool that I am using for local dev to test things out. We don't actually use it in production anywhere.

Reply:
ah yeah I think you've told me this before ....I need to try and remember and pay attention to "adhoc" being in the file name and that it means it is non-production use